components/slidingPanel/slidingPanel.js

```diff
@@ -68,13 +89,27 @@ export default class SlidingPanel extends Component {
     active: PropTypes.bool,
 
     /**
+     * Defines the width of the panel.
+     */
+    width: PropTypes.string,
```

Comment:
why do we need this prop?

Reply:
we have different width for panels (I found at least 2 different types). If we will have this prop, we will not need to override the width in styles